meteor-app/imports/ui/actions/workspace_switch_welcome_window/index.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+export const WORKSPACE_SWITCH_WELCOME_WINDOW = {
+  type: 'WORKSPACE_SWITCH_WELCOME_WINDOW',
+  payloadSchema: {
+    value: PropTypes.bool,
```

Comment:
This action doesn't need any input at all, as it simply flips the state value. In another design where you could have an optional value is where the value enforces the final result. Consider the following examples:

```JavaScript
$ele.toggleClass("someClassName"); // Removes if present, adds otherwise.
$ele.toggleClass("someClassName", true); // Adds if not present, do nothing otherwise.
```